### PR TITLE
Add scan script and exceptions file to master branch

### PR DESCRIPTION
**Pull request by kirthik**
Copied the files from `AutoRest` branch to `master` branch

### CODE REVIEW

**Comment by azurecla** [comment]
Hi __@kirthik__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <p>
        It looks like you're working at Microsoft (kirthik). If you're full-time, we DON'T require a contribution license agreement.
    </p>
    <p>
        If you are a vendor, DO please sign the electronic contribution license agreement. It will take 2 minutes and there's no faxing! https://cla.azure.com.
    </p>

TTYL, AZPRBOT;

File: tools/RunCredScan.ps1 [hunk 1]
```diff
@@ -0,0 +1,54 @@
+﻿$credentialScannerLocation = $args[0]+"\CredentialScanner.exe"
```
Comment (markcowl) [inline tools/RunCredScan.ps1]: Can we put a comment at the top of the file with an example of usage with all parameters?

File: tools/RunCredScan.ps1 [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+}
+
+Write-Host "Running credential scanner from location: " $credentialScannerLocation
+Write-Host "Source location for credential scanner: " $sourceLocation
+Write-Host "Searcher file location for credential scanner: " $searcherFile
+
+$proc = Start-Process -FilePath $credentialScannerLocation -ArgumentList $sourceLocation, $searcherFile, $logFile -NoNewWindow -Wait -PassThru
+
+#Check process exit code after running credential scanner
+if ($proc.ExitCode -ne 0) {
```
Comment (markcowl) [inline tools/RunCredScan.ps1]: (1) Will $Error contain useful information if there is an error?

File: tools/RunCredScan.ps1 (hunk 1 above)
Comment (kirthik) [inline tools/RunCredScan.ps1]: Sure will do.

File: tools/RunCredScan.ps1 (hunk 2 above)
Comment (kirthik) [inline tools/RunCredScan.ps1]: Yes, it will.

**Pull request merged**